python/ray/remote_function.py

```diff
@@ -85,69 +76,31 @@ def __init__(
         language,
         function,
         function_descriptor,
-        num_cpus,
-        num_gpus,
-        memory,
-        object_store_memory,
-        resources,
-        accelerator_type,
-        num_returns,
-        max_calls,
-        max_retries,
-        retry_exceptions,
-        runtime_env,
-        placement_group,
-        scheduling_strategy: SchedulingStrategyT,
+        option_dict,
     ):
         if inspect.iscoroutinefunction(function):
             raise ValueError(
                 "'async def' should not be used for remote tasks. You can wrap the "
                 "async function with `asyncio.get_event_loop.run_until(f())`. "
                 "See more at https://docs.ray.io/en/latest/ray-core/async_api.html#asyncio-for-remote-tasks"  # noqa
             )
+        self._option_dict = option_dict
+
+        # TODO(suquark): This is a workaround for class attributes of options.
+        # They are being used in some tests. Need cleanup later.
```

Comment:
Actually not just tests, for example `_max_calls` is used in function_manager.py